can/interfaces/vector/canlib.py

```diff
@@ -96,18 +102,46 @@ def __init__(self, channel, can_filters=None, poll_interval=0.01,
 
         permission_mask = vxlapi.XLaccess()
         # Set mask to request channel init permission if needed
-        if bitrate:
+        if bitrate or fd:
             permission_mask.value = self.mask
-        vxlapi.xlOpenPort(self.port_handle, self._app_name, self.mask,
-                          permission_mask, rx_queue_size,
-                          vxlapi.XL_INTERFACE_VERSION, vxlapi.XL_BUS_TYPE_CAN)
+        if fd:
+            vxlapi.xlOpenPort(self.port_handle, self._app_name, self.mask,
+                              permission_mask, 16000,
```

Comment:
Why 16000? Can this also be `rx_queue_size`?

Reply:
Yeah, hardcoded value was taken over from a Vector example. Anyway the Vector documentation gives different ranges for rx_queue_size in case of CAN or CAN-FD. So I will change the default value to be in both ranges and use rx_queue_size in both cases.

Reply:
Hello together,

did somebody test the CAN FD ? Is it working? 

Thanks

David